helm-chart/values.yaml.changelog.md

```diff
@@ -6,7 +6,20 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
-## Unreleased (breaking changes)
+
+## Upgrading to Renku 0.8.x (Unreleased)
+* NEW *notebooks.serverDefaults* has been added with default values that will be
```

Comment:
I'd try to keep this short and just link to the new section in the chart readme that you also add: https://github.com/SwissDataScienceCenter/renku/tree/master/helm-chart#upgrading-to-08x (I know it's a bit annoying to make those links ahead of time for unreleased tags)